x-pack/plugins/data_visualizer/public/application/index_data_visualizer/utils/saved_search_utils.ts

```diff
@@ -168,13 +174,14 @@ export function getEsQueryFromSavedSearch({
     };
   }
 
+  // @TODO: remove
```

Comment:
Is this TODO needed, and the comment out line below?

Reply:
Removed here [`265d633` (#149459)](https://github.com/elastic/kibana/pull/149459/commits/265d6332a292c0281d2d91754b89d52764e4abeb)